runtime/near-vm-logic/src/logic.rs

```diff
@@ -887,10 +887,17 @@ impl<'a> VMLogic<'a> {
         let receiver_id = self.read_and_parse_account_id(receiver_id_ptr, receiver_id_len)?;
         let method_names = Self::memory_get(self.memory, method_names_ptr, method_names_len)?;
         // Use `,` separator to split `method_names` into a vector of method names.
-        let method_names = method_names
-            .split(|c| *c == b',')
-            .map(|v| if v.is_empty() { Err(HostError::EmptyMethodName) } else { Ok(v.to_vec()) })
-            .collect::<Result<Vec<_>>>()?;
+        let method_names =
```

Comment:
Do you have https://github.com/nearprotocol/nearcore/blob/master/rustfmt.toml in `~/.config/rustfmt/` ?

Reply:
`Err(HostError::EmptyMethodName.into())` changed all formatting

Reply:
I'm unsure of my rustfmt config. Is ~/.config/rustfmt a must to make reformat?

From here: https://github.com/rust-lang/rustfmt/blob/master/Configurations.md
Looks like project rustfmt.toml will take precedence